google-storage/test/google-contetnt-provider.test.ts

```diff
@@ -0,0 +1,70 @@
+import { MockStorage, MockBucket, MockFile } from '../test/mockstorage'
+import { Observable } from "rxjs";
+
+describe("get", () => {
```

Comment:
This tests are testing the mocks, not the provider.

We should be testing the provider here and using the mocks to stub out functionality in the provider.